transactions/epoch/admin/update_epoch_config.cdc

```diff
@@ -0,0 +1,22 @@
+import FlowEpoch from 0xEPOCHADDRESS
+
+transaction(newStakingViews: UInt64, newPhaseViews: UInt64, newEpochViews: UInt64) {
```

Comment:
A couple suggestions:
* make the argument order consistent with that used in the existing [service account transaction](https://github.com/onflow/service-account/blob/main/templates/adjust_epoch_length.cdc#L3)
* include `dkg` in the `DKG Phase Length` variable name (all three variables represent a phase)
```suggestion
transaction(newDKGViews: UInt64, newStakingViews: UInt64, newEpochViews: UInt64) {
```

Reply:
https://github.com/onflow/flow-core-contracts/pull/282/commits/c58ed63eddd13fdacde845bdf7120038bff021ad